modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -0,0 +1,54 @@
+// This is included in the following assemblies:
+//
+// networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="creating-re-encrypt-route-with-custom-certificate_{context}"]
+= Creating an Ingress route with a custom certificate
+The `route.openshift.io/destination-ca-certificate-secret` annotation can be used to define an Ingress route with a custom certificate (CA).
+
+.Prerequisites
+* You must have a certificate/key pair in PEM-encoded files, where the certificate is valid for the route host.
+* You may have a separate CA certificate in a PEM-encoded file that completes the certificate chain.
+* You must have a separate destination CA certificate in a PEM-encoded file.
+* You must have a service that you want to expose.
+
+
+.Procedure
+
+. Add the `route.openshift.io/destination-ca-certificate-secret` to the Ingress object annotations:
```

Comment:
i would prefer ` Add the `route.openshift.io/destination-ca-certificate-secret` to the Ingress  annotations`